consul/config/basic_config.json

```diff
@@ -2,8 +2,8 @@
   "datacenter": "{{cfg.server.datacenter}}",
   "data_dir": "{{cfg.server.data-dir}}",
   "log_level": "{{cfg.server.loglevel}}",
-  "bind_addr": "{{cfg.server.bind}}",
```

Comment:
Please remove the references to server and client bind from the default.toml since they are no longer needed.